pkg/storage/accessibility_request_test.go

```diff
@@ -0,0 +1,56 @@
+package storage
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"time"
+
+	"github.com/facebookgo/clock"
+
+	"github.com/cmsgov/easi-app/pkg/testhelpers"
+)
+
+func (s StoreTestSuite) TestFetchAccessibilityRequestMetrics() {
+	ctx := context.Background()
+
+	mockClock := clock.NewMock()
+	settableClock := testhelpers.SettableClock{Mock: mockClock}
+	s.store.clock = &settableClock
+
+	intake := testhelpers.NewSystemIntake()
+	_, err := s.store.CreateSystemIntake(ctx, &intake)
+	s.NoError(err)
+
+	// create a random year to avoid test collisions
+	// uses postgres max year minus 1000000
+	rand.Seed(time.Now().UnixNano())
```

Comment:
If we assume an empty test database at the beginning of each run, do we need this? Having something in the tests be random means that it's very hard to reproduce failures should they occur.

Reply:
IF we assume an empty database, yes. We run all the tests together, so we need to separate out these times from the others